Integrations/python/setup.py

```diff
@@ -55,6 +55,6 @@ def normalize_version(ver):
       keywords='Deephaven Development',
       packages=find_packages(exclude=['docs', 'test']),
       install_requires=['deephaven-jpy=={}'.format(__normalized_version__),
-                        'numpy', 'dill>=0.2.8', 'wrapt', 'pandas', 'numba;python_version>"3.0"',
-                        'enum34;python_version<"3.4"'],
+        'numpy', 'torch', 'torchsummary', 'torchvision', 'scikit-learn==0.23.1', 'dill>=0.2.8', 'optuna',
+        'ray', 'wrapt', 'pandas', 'tensorflow', 'numba;python_version>"3.0"', 'enum34;python_version<"3.4"'],
```

Comment:
Add your changes to the end of the list so that it is easier for the devs to know what is the internal stuff vs the quanty stuff.